src/lib/is-hostname.js

```diff
@@ -0,0 +1,11 @@
+// checks whether the hostname has been entered into the format that Nori expects, because Nori then interpolates it into the string it uses when calling Tako.
+
+const isHostname = hostname => {
+	// check if totally mad
+	if (/(https:\/\/)*(.*)\/tako(.*)/.test(hostname)) {
+		return /(https:\/\/)*(.*)\/tako(.*)/g.exec(hostname)[2]
+	}
+	return 'Please enter the hostname that sits within `https://.../tako`'
```

Comment:
this isn't a super clear validation message. maybe something like `Please enter a valid Tako hostname, e.g. "customer-products-tako.in.ft.com"`

Reply:
🆗